docs/es/week03/03-1.md

```diff
@@ -80,13 +80,13 @@ $$
 These formulas are applied in a matrix form. Note that the dimensions of the terms should be consistent. The dimension of $u$,$w$,$\frac{\partial H}{\partial u}^\top$,$\frac{\partial C}{\partial w}^\top$ are $[N_u \times 1]$,$[N_w \times 1]$,$[N_u \times N_w]$,$[N_w \times 1]$, respectively. Therefore, the dimension of our backpropagation formula is consistent.
 -->
 
-Estas fórmulas son aplicadas en forma matricial. Nótese que las dimensiones de los términos deben de ser consistentes. La dimensión de $u$,$w$,$\frac{\partial H}{\partial u}^\top$,$\frac{\partial C}{\partial w}^\top$ son $[N_u \times 1]$,$[N_w \times 1]$,$[N_u \times N_w]$,$[N_w \times 1]$, respectivamente. Por consiguiente, la dimensión de la propagación hacia atrás es consistente.
+Estas fórmulas se aplican en forma matricial. Presta atención a que las dimensiones de los términos deben de ser consistentes. La dimensión de $u$, $w$, $\frac{\partial H}{\partial u}^\top$, $\frac{\partial C}{\partial w}^\top$ son $[N_u \times 1]$, $[N_w \times 1]$, $[N_u \times N_w]$, $[N_w \times 1]$, respectivamente. Por consiguiente, la dimensión de la propagación hacia atrás es consistente.

```

Comment:
suggestion: "**Presta**..." to "**Prestemos**.."  (tense homogeneity with the rest of the sections: example line 69 and 142)

Reply:
I don't agree. First, it's not the *tense* being changed with your suggestion but the *mood*. Second, the original it's in the imperative "Note that the..." and the other lines being referred (i.e., 69 and 142) are not; they're in the present tense in the indicative mood in the original and in the translation.

Reply:
OK

Marking the pull as "approved", as the (few) minor errors were fixed.
 Thank you for your time and for sharing your knowledge!